backend.native/tests/interop/objc/tests/KT38234_override.h

```diff
@@ -0,0 +1,9 @@
+#import <Foundation/NSObject.h>
+
+@protocol P1
```

Comment:
Currently all tests in this directory are compiled together, so using more unique names (like in [original reproducer](https://github.com/JetBrains/kotlin-native/compare/kt-38234#diff-62b02828c319a01dde4c977ea7a7cab2R3)) is preferable.